qos-monitor/src/main/resources/application.properties

```diff
@@ -26,17 +26,17 @@ domain.port=8451
 ############################################
 
 # Name of the core system
-core_system_name=QOSMONITOR
+core_system_name=qosmonitor
 
 # Show all request/response in debug log
-log_all_request_and_response=false
+log_all_request_and_response=true
 
 # Service Registry web-server parameters (to register the QoS Monitor services)
 sr_address=127.0.0.1
 sr_port=8443
 
 # Gatekeeper core system is present
-gatekeeper_is_present=true
+gatekeeper_is_present=false
```

Comment:
should be true when we merge into development

Reply:
done